frontend/test/resolution/testArrays.cpp

```diff
@@ -138,6 +141,7 @@ module M {
   }
 
   assert(guard.errors().size() == 0);
+  guard.realizeErrors(context);
```

Comment:
Hah, I think it's because I didn't know about `countWarnings` once and got a complaint when invoking `realizeErrors` in the debugger, and just assumed it wanted a context, and it took one since it can coerce to bool. Will fix